docs/developer-docs/smart-contracts/overview/canister-lifecycle.mdx

```diff
@@ -0,0 +1,63 @@
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+
+# Canister lifecycle
+
+<MarkdownChipRow labels={["Beginner", "Concept"]} />
+
+## Write
+
+The ICP accepts and executes canister code in the WebAssembly (Wasm) binary format.
+Usually, developers write code in a higher-level language and then compile the code into a Wasm binary that can be deployed into a canister.
+
+## Deploy
+
+### Create
+
+A developer can create an empty canister on ICP by calling the system canister.
+It can be done using the `NNS frontend dapp` in the browser or `dfx` in the command line.
```

Comment:
```suggestion
It can be done using the [NNS frontend dapp](/docs/current/developer-docs/daos/nns/nns-app-quickstart#deploying-and-managing-canisters-from-the-nns-dapp) in the browser or [`dfx`](/docs/current/developer-docs/developer-tools/cli-tools/cli-reference/) in the command line.
```